server/src-lib/Hasura/Server/App.hs

```diff
@@ -539,6 +545,23 @@ configApiGetHandler serverCtx@ServerCtx{..} consoleAssetsDir =
                 (EL._lqsOptions $ scLQState) consoleAssetsDir
       return $ JSONResp $ HttpResponse (encJFromJValue res) []
 
+gqlPostApiHandler 
+  ::  ( HasVersion
+      , MonadIO m
+      , UserAuthentication (Tracing.TraceT m)
+      , HttpLog m
+      , Tracing.HasReporter m
+      , E.MonadGQLExecutionCheck m
+      , MonadQueryLog m
+      , GH.MonadExecuteQuery m
+      , EQ.MonadQueryInstrumentation m
+      )
```

Comment:
I think its a good idea to add these constraints to the `runGQLApiHandler` function as well.